browser/src/control/Control.DocumentRepair.js

```diff
@@ -2,89 +2,116 @@
 /*
  * L.Control.DocumentRepair.
  */
-/* global _ _UNO */
+/* global _ */
+/* _UNO */
 L.Control.DocumentRepair = L.Control.extend({
-	options: {
-		position: 'topright'
-	},
-
-	initialize: function (options) {
-		L.setOptions(this, options);
-	},
-
-	onAdd: function () {
-		this._initLayout();
-
-		return this._container;
-	},
-
-	_initLayout: function () {
-		this._container = L.DomUtil.create('div', 'leaflet-control-layers');
-		this._container.style.visibility = 'hidden';
-
-		var closeButton = L.DomUtil.create('a', 'leaflet-popup-close-button', this._container);
-		closeButton.href = '#close';
-		closeButton.innerHTML = '&#215;';
-		L.DomEvent.on(closeButton, 'click', this._onCloseClick, this);
-
-		var wrapper = L.DomUtil.create('div', 'leaflet-popup-content-wrapper', this._container);
-		var content = L.DomUtil.create('div', 'leaflet-popup-content', wrapper);
-		var labelTitle = document.createElement('span');
-		labelTitle.innerHTML = '<b>' + _('Repair Document') + '</b>';
-		content.appendChild(labelTitle);
-		content.appendChild(document.createElement('br'));
-		content.appendChild(document.createElement('br'));
-		var table = L.DomUtil.create('table', '', content);
-		var tbody = this._tbody = L.DomUtil.create('tbody', '', table);
-		var tr = L.DomUtil.create('tr', '', tbody);
-		var th = L.DomUtil.create('th', '', tr);
-		L.DomUtil.setStyle(th, 'display', 'none');
-		th.appendChild(document.createTextNode(''));
-		th = L.DomUtil.create('th', '', tr);
-		th.appendChild(document.createTextNode(_('Type')));
-		th = L.DomUtil.create('th', '', tr);
-		th.appendChild(document.createTextNode(_('Index')));
-		th = L.DomUtil.create('th', '', tr);
-		th.appendChild(document.createTextNode(_('Comment')));
-		th = L.DomUtil.create('th', '', tr);
-		th.appendChild(document.createTextNode(_('User name')));
-		th = L.DomUtil.create('th', '', tr);
-		th.appendChild(document.createTextNode(_('Timestamp')));
+	builder: null,
+	actions: null,
+	selected: null,
 
-		var inputButton = document.createElement('input');
-		inputButton.type = 'button';
-		inputButton.value = _('Jump to state');
-		L.DomEvent.on(inputButton, 'click', this._onJumpClick, this);
-		content.appendChild(document.createElement('br'));
-		content.appendChild(document.createElement('br'));
-		content.appendChild(inputButton);
+	addTo: function (map) {
+		this.remove();
+		/*
+		this.builder = L.control.jsDialogBuilder({
```

Comment:
some leftover?

Reply:
> some leftover?

yep

I also still need to fix cypress tests, but thanks for reminding me